apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html

```diff
@@ -689,6 +689,42 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                                 placeholder="Description"></textarea>
                         </div>
                     </div>
+
+                    <ng-template>
+                        <div class="clearfix adjust-receipt font-14">
+                            <p class="bold">Adjusted Receipts <a href="javascript:void(0)"><i class="icon-edit11 font-10"></i></a></p>
+                            <p class="font-12 text-light mb-1">₹2400.00</p>
+                            <div class="clearfix adjustment-filed">
```

Comment:
Use `adjustment-field` in class name

Reply:
class name changed  adjustment-filed to adjustment-field